Rocket.Chat/Managers/AppManager.swift

```diff
@@ -129,14 +127,15 @@ extension AppManager {
         func openDirectMessage() -> Bool {
             guard let directMessageRoom = Subscription.find(name: username, subscriptionType: [.directMessage]) else { return false }
 
-            let controller = ChatViewController.shared
-            controller?.subscription = directMessageRoom
+            // TODO: Have to fix here
+            // let controller = ChatViewController.shared
```

Comment:
Regression: Will not open direct message with user when pressing notification
Regression: Will not open direct message with user when pressing "conversation" in user action sheet

Reply:
✅